software/scripts/i2c0.func

```diff
@@ -1,7 +1,6 @@
 #!/bin/sh
 
-. ./bit.func
-
+. ../scripts/bit.func
```

Comment:
unnecessary change

Reply:
This was required by `prep_icsp.sh` .It includes `i2c0.func` which further includes `bit.func` ,so if the  `prep_icsp.sh`  is run , it can't find `bit.func`.I would have liked to do something more generic but couldn't find anything

Reply:
I see, for that you can use something like `cd $(dirname $(realpath $0))`

Reply:
I couldn''t find a proper way to do that ,once way was to use `cd $(dirname $BASH_SOURCE)` in the `i2c0.func` file

Reply:
what's the problem with the line i suggested?

Reply:
`cd $(dirname $(realpath $0))` lets us switch to the path of the source script (`bringup-scripts`),but does not help in finding the path of `bit.func`

Reply:
is does, when putting `cd $(dirname $(realpath $0))` inside `i2c0.func`

Reply:
Yes this would work ,but we would have to` cd `back to the `bringup-scripts` folder after sourcing the `i2c0.func` file.
If we are going with this should I change `shebang ` to `bash` for all the scripts or only one which are required

Reply:
yes, of course, `pushd` and `popd` should work fine for that